charts/preview/templates/operator-role.yaml

```diff
@@ -32,6 +32,12 @@ rules:
       - get
       - delete
       - update
+  - apiGroups:
```

Comment:
This is in here twice. See line 41. Intentional?

Reply:
Correct, Different verbs for different resources